docker-app/qfieldcloud/core/urls.py

```diff
@@ -105,4 +105,24 @@
         deltas_views.ListDeltasByDeltafileView.as_view(),
     ),
     path("deltas/apply/<uuid:projectid>/", deltas_views.ApplyView.as_view()),
+    path(
```

Comment:
Where this comes from? This PR is about adding new fields to a model, not adding new endpoints.

Reply:
Fixed by uploading the original files